.github/workflows/test.yml

```diff
@@ -167,7 +167,7 @@ jobs:
           SKIP_GIT: true
           GITHUB_CONTEXT: ${{ toJson(github) }}
 
-      - run: pnpm run test-notypes && pnpm run test:functional
+      - run: pnpm run test-notypes && pnpm run test:functional && pnpm run test:memory
```

Comment:
If we plan on adding many tests, would it make sense to extract it into its own job?

Reply:
Related: It would be nice if we could plug in existing tests automatically somehow, but that might require much more work (eg. injecting `describe`, `test`, etc with esbuild to get a jest-like "runnable" suite).

Reply:
It might be, they are not the fastest, since they are running with optimizing compiler disabled.